plugins/thefuck/thefuck.plugin.zsh

```diff
@@ -0,0 +1,17 @@
+if [[ -z $commands[thefuck] ]]; then
+    echo 'thefuck is not installed, you should "pip install thefuck" first'
+    return -1
+fi
+
+# Register alias
+eval "$(thefuck-alias)"
+
+fuck-command-line() {
+    FUCK="$(THEFUCK_REQUIRE_CONFIRMATION=0 thefuck $(fc -ln -1 | tail -n 1) 2> /dev/null)"
```

Comment:
Could this `$FUCK` be made local? Or is it intended to stick around for reference?